src/grt/grt-avhpi.adb

```diff
@@ -126,14 +139,18 @@ package body Grt.Avhpi is
             end case;
          when VhpiIndexedNames =>
             case Ref.Kind is
-               when VhpiGenericDeclK =>
+               when VhpiGenericDeclK |
+                    VhpiPortDeclK |
+                    VhpiSigDeclK =>
                   Res := (Kind => AvhpiNameIteratorK,
                           Ctxt => Ref.Ctxt,
+                          -- What does the 'N_' indicate?
```

Comment:
It's the prefix for names in the vhpiHandleT.  All the names in a record must be unique.